pkg/rancher-desktop/config/settings.ts

```diff
@@ -297,6 +298,14 @@ export function load(): Settings {
     if (os.platform() === 'linux' && !process.env['APPIMAGE']) {
       settings.application.updater.enabled = false;
     }
+
+    const appVersion = getProductionVersion();
+
+    // Auo-update doesn't work for CI or local builds, so don't enable it by default
+    if (appVersion.includes('-') || appVersion.includes('?')) {
+      settings.application.updater.enabled = false;
```

Comment:
Just realized that we're actually modifying `defaultSettings` in place here (we did that already, setting the VM memory size etc). Would it make sense to have line 288 be `settings = clone(defaultSettings)` instead?

Reply:
Yes, I think it does. I've amended the commit.